### ISSUE TITLE
Module has no exported member "Bucket" in typescript project

### ISSUE THREAD

**Issue by mlejva** [googleapis/nodejs-storage#433]
Thanks for stopping by to let us know something could be better!

Please run down the following list and make sure you've tried the usual "quick
fixes":

  - Search the issues already opened: https://github.com/googleapis/nodejs-storage/issues
  - Search the issues on our "catch-all" repository: https://github.com/GoogleCloudPlatform/google-cloud-node
  - Search StackOverflow: http://stackoverflow.com/questions/tagged/google-cloud-platform+node.js

If you are still having issues, please be sure to include as much information as
possible:

#### Environment details

  - OS: macOS 10.14
  - Node.js version: 10.6.0
  - npm version: 6.4.1
  - `@google-cloud/storage` version: 2.0.3

#### Steps to reproduce

  1. import Storage object
```
import { Storage } from '@google-cloud/storage';
```
  2. run `tsc` to convert Typescript into Javascript
This will result in following error
```
node_modules/firebase-admin/lib/index.d.ts(18,9): error TS2305: Module '"/node_modules/@google-cloud/storage/build/src/index"' has no exported member 'Bucket'.
```

This issue is also opened in [firebase-admin-node repo](https://github.com/firebase/firebase-admin-node/issues/358)

I'm happy to fix this issue but I'm not sure whether `Bucket` class should be exported from the index file and the issue is caused by this repo. Or the issue is caused by firebase-admin-node repo because of incorrect usage.

Following these steps will guarantee the quickest resolution possible.

Thanks!

**Comment by otnielgomez** [comment]
As a work arround in your /node_modules/firebase-admin/lib/index.d.ts replace:

**Line 18**
Instead of:
import {Bucket} from '@google-cloud/storage';

Use: 
import {Storage} from '@google-cloud/storage';

**AND Line 589**

Instead of:
bucket(name?: string): Bucket;

Use: 
bucket(name?: string): Storage;


It worked for me, im using: 

"@google-cloud/storage": "^2.0.3",
"firebase-admin": "~5.12.1",
"firebase-functions": "^2.0.5"

**Comment by JustinBeckwith** [comment]
Fixed in #460
